### PR TITLE
Update sandman_de.ts and fix spelling mistakes in NewBoxWizard.cpp

### PR DESCRIPTION
**Pull request by bastik-1001**
Fresh fork to update the translation and fix spelling mistakes.

### CODE REVIEW

**Comment by bastik-1001** [comment]
I wonder if it would be smart to prevent the bot from updating the language files, whenever the spell check has found some errors.

Is it the fault of Qt that certain strings change to something totally different? A string about process tokens changes to a string about the box being discarded. Even if the order changes, A does not become B or vice versa.

File: SandboxiePlus/SandMan/sandman_de.ts [hunk 1]
```diff
@@ -67,21 +67,17 @@
...
     </message>
     <message>
         <location filename="Wizards/NewBoxWizard.cpp" line="481"/>
         <source>This feature may reduce compatybility as it also prevents box located processes from writing to host located once and even starting them.</source>
-        <translation type="unfinished"></translation>
-    </message>
-    <message>
-        <source>Use a Sandboxie login instead of an anonymous token (experimental)</source>
-        <translation type="vanished">Verwende einen Sandboxie-Login anstelle eines anonymen Token (experimentell)</translation>
+        <translation>Diese Funktion könnte die Kompatibilität einschränken, da sie Prozesse, die sich in der Sandbox befinden, daran hindert solche des Hosts zu schreiben und diese sogar zu starten.</translation>
```
Comment (bastik-1001) [inline SandboxiePlus/SandMan/sandman_de.ts:80]: While writing that, I changed the structure of the translation many times, never having it sound as nice as I wanted it to. If someone has a better way of saying this, feel free to change it.

File: SandboxiePlus/SandMan/Wizards/NewBoxWizard.cpp [hunk 2]
```diff
@@ -478,7 +478,7 @@ CAdvancedPage::CAdvancedPage(QWidget *parent)
     registerField("boxToken", pBoxToken);
 
     QCheckBox* pImageProtection = new QCheckBox(tr("Prevent sandboxes programs installed on host from loading dll's from the sandbox"));
-    pImageProtection->setToolTip(tr("This feature may reduce compatybility as it also prevents box located processes from writing to host located once and even starting them."));
+    pImageProtection->setToolTip(tr("This feature may reduce compatibility as it also prevents box located processes from writing to host located once and even starting them."));
```
Comment (offhub) [inline SandboxiePlus/SandMan/Wizards/NewBoxWizard.cpp:481]: once -> ones

Comment (bastik-1001) [reply SandboxiePlus/SandMan/Wizards/NewBoxWizard.cpp]: Thank you, I updated my changes to include that fix to the grammar.

**Pull request merged**